pyodide-build/pyodide_build/io.py

```diff
@@ -75,7 +76,19 @@ def _check_patches_extra(cls, values: dict[str, Any]) -> dict[str, Any]:
         return values
 
 
+class _EnumExports(str, Enum):
+    pyinit = "pyinit"
+    requested = "requested"
+    whole_archive = "whole_archive"
+
+
 _BuildSpecExports = Literal["pyinit", "requested", "whole_archive"]
+
+
+def _convert_exports(e: _EnumExports | None) -> _BuildSpecExports | None:
+    return cast(_BuildSpecExports | None, e)
+
```

Comment:
This is awkward but we need it until typer merges something like this: https://github.com/tiangolo/typer/pull/429